services-camel/src/main/java/edu/unc/lib/dl/services/camel/BinaryMetadataProcessor.java

```diff
@@ -45,10 +47,7 @@
  *
  */
 public class BinaryMetadataProcessor implements Processor {
-
     private final int BINARY_PATH_DEPTH = 3;
-    private final int BINARY_PATH_LENGTH = 2;
-
     private String baseBinaryPath;
 
     protected BinaryMetadataProcessor(String baseBinaryPath) {
```

Comment:
should this constructor really have protected access? other processors look like they have public constructors

Reply:
fixed